### PR TITLE
Simplify platform specific instructions

### PR DESCRIPTION
**Pull request by Computr0n**
https://github.com/JuulLabs/kable/issues/70

### CODE REVIEW

**Comment by Computr0n** [comment]
Haha, I see we both made a PR to address this at the same time. Do as you wish.

**Comment by twyatt** [comment]
> Haha, I see we both made a PR to address this at the same time. Do as you wish.

I just noticed that too. 😄 

I was overthinking it, I like your approach better (it's simpler).

**Review by twyatt — APPROVED** [review]

**Comment by twyatt** [comment]
@Computr0n thanks for all the help testing and figuring out the misinformation in the README, and then also contributing updates to the documentation. I'll merge once I get sufficient approvals from the team.

**Review by sdonn3 — APPROVED** [review]

**Pull request merged**
